packages/ui/multiwallet-ui/src/index.tsx

```diff
@@ -0,0 +1,438 @@
+import React, { HTMLAttributes, useCallback, useEffect } from 'react';
+import {
+  Box,
+  ButtonBase,
+  IconButton,
+  Modal,
+  Paper,
+  PaperProps,
+  Typography,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import CloseIcon from '@material-ui/icons/Close';
+import { ConnectorInterface } from '@renproject/multiwallet-base-connector';
+import { useMultiwallet } from './MultiwalletProvider';
+import { RenNetwork } from '@renproject/interfaces';
+
+export * from './MultiwalletProvider';
+
+export interface ConnectorConfig<P, A> {
+  /**
+     Name of the wallet
+   */
+  name: string;
+  /**
+     URL for logo to be shown (might change in future to a component)
+   */
+  logo: string;
```

Comment:
I think a few integrators will avoid using urls for logs, to avoid the loading lag. This is already compatible with the image import used by `create-react-app` (which imports the image as a base64 data url) but it might be worth considering other ways of providing logos, such as JSX elements or `create-react-app`'s SVG elements.

At the very least, I think that if a logo is provided, the img tag's height should be fixed to avoid it changing when the url is loaded.